waspc/src/Wasp/Generator/ServerGenerator/AuthG.hs

```diff
@@ -40,6 +41,19 @@ genCoreAuth auth = C.makeTemplateFD tmplFile dstFile (Just tmplData)
               "userEntityLower" .= Util.toLowerFirst userEntity
             ]
 
+genAuthMiddleware :: Wasp.Auth.Auth -> FileDraft
```

Comment:
Ha nice job :D! A bit boilerplatish, this whole file, but nothing we should worry about right now, it is not worth the effort at the moment.